subset/password/brute_test/src/main/java/PackageManager.java

```diff
@@ -0,0 +1,119 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+public class PackageManager {

```

Comment:
This class doesn't look like a "PackageManager" -- please consider an appropriate name.